spec/spec_helper.rb

```diff
@@ -8,8 +8,6 @@
 
 require 'voxpupuli/test/spec_helper'
 
-add_mocked_facts!
```

Comment:
This was added manually. It may break things.

Reply:
the unit tests still pass, so that looks fine.